repeat/p5js/hidden-circles/index.js

```diff
@@ -0,0 +1,69 @@
+var dis = 20;
+
+function setup() {
+  createCanvas(500, 500);
+  background(255);
+}
+
+function drawLines(count, distance) {
+  distance = distance || 20;
+  
+  for (var i = 0; i < count; i++) {
+     var x = i * distance;
+    
+    line(x, 0, x, 500);
+  }
+}
+
+function draw() {
+  stroke(0);
+  strokeWeight(2);
+  drawLines(100, dis);
+  
+  strokeWeight(10);
+  stroke(255, 255, 255);
+  noFill();
+  ellipse(250, 250, 100);
```

Comment:
Please use DRY code. Don't repeat yourself. The ellipses below can be done in a single loop. I also would add the "Law of Closure" tag.